modules/update-service-create-service-cli.adoc

```diff
@@ -0,0 +1,80 @@
+[id="update-service-create-service-cli_{context}"]
+= Creating an OpenShift Update Service application by using the CLI
+
+You can use the {product-title} CLI to create an OpenShift Update Service application.
+
+.Prerequisites
+
+* The OpenShift Update Service Operator has already been installed.
+* The Openshift Update Service graph-data container image has already been created and pushed to a repository accessible to the OpenShift Update Service.
+* The current release and update target releases have already been mirrored to a locally accessible registry.
+
+.Procedure
+
+To create an OpenShift Update Service application by using the {product-title} CLI:
+
+. Configure the OpenShift Update Service target namespace, for example, `openshift-update-service`:
++
+[source,terminal]
+----
+$ NAMESPACE=openshift-update-service
+----
++
+The namespace must match the `targetNamespaces` value from the OperatorGroup.
+
+. Configure the name of the OpenShift Update Service application, for example, `service`:
++
+[source,terminal]
+----
+$ NAME=service
+----
+
+. Configure the local registry and repository for the release images as configured in xref:modules/update-service-mirror-release.adoc#update-service-mirror-release_{context}[Mirroring the OpenShift Container Platform image repository], for example, `registry.example.com/ocp4/openshift4-release-images`:
++
+[source,terminal]
+----
+$ RELEASE_IMAGES=registry.example.com/ocp4/openshift4-release-images
+----
+
+. Set the local pullspec for the graph-data image to the graph-data container image created in xref:modules/update-service-graph-data.adoc#update-service-graph-data_{context}[Creating the OpenShift Update Service graph data container image], for example, `registry.example.com/openshift/graph-data:latest`:
++
+[source,terminal]
+----
+$ GRAPH_DATA_IMAGE=registry.example.com/openshift/graph-data:latest
+----
+
+. Create an OpenShift Update Service application object:
++
+[source,terminal]
+----
+$ oc -n "${NAMESPACE}" create -f - <<EOF
+apiVersion: updateservice.operator.openshift.io/v1
+kind: UpdateService
+metadata:
+  name: ${NAME}
+spec:
+  replicas: 2
+  releases: ${RELEASE_IMAGES}
+  graphDataImage: ${GRAPH_DATA_IMAGE}
+EOF
+----
+
+. Verify the OpenShift Update Service application:
+
+.. Use the following command to obtain a policy engine route:
++
+[source,terminal]
+----
+$ while sleep 1; do POLICY_ENGINE_GRAPH_URI="$(oc -n "${NAMESPACE}" get -o jsonpath='{.status.policyEngineURI}/api/upgrades_info/v1/graph{"\n"}' updateservice "${NAME}")"; SCHEME="${POLICY_ENGINE_GRAPH_URI%%:*}"; if test "${SCHEME}" = http -o "${SCHEME}" = https; then break; fi; done
+----
++
+You might need to poll until the command succeeds.
+
+.. Retrieve a graph from the policy engine. Be sure to specify a valid version for *channel*. For example, if running in {product-title} {product-version} use version {product-version}, e.g. *stable-{product-version}*:
```

Comment:
* channel and stable-{product-version} should be in backticks instead of bold/asterisks.
* The second sentence has too many example/e.g. I'd shorten to:
```
For example, if running in {product-title} {product-version}, use `stable-{product-version}`:
```